tests/components/airzone/test_sensor.py

```diff
@@ -20,7 +20,7 @@ async def test_airzone_create_sensors(
 
     # Zones
     state = hass.states.get("sensor.despacho_temperature")
-    assert state.state == "21.2"
+    assert state.state == "21.20"
```

Comment:
Why is this test affected?

Reply:
It's affected because the sensor's native value is `70.16 °F`, which is exactly 21.2 °C.
The value before conversion is assumed to have 2 decimals, which is preserved after automatic unit conversion from °F to °C.

Reply:
Sounds good.